docs/contributing/documentation.rst

```diff
@@ -20,27 +20,37 @@ your work.
 
 Steps for contributing documentation:
 
+-  Follow the workflow for :ref:`contributing` to Anemoi, including
+   creating an issue and submitting a Pull Request.
+
 -  Identify the most appropriate place for your addition (e.g., overall
    `Anemoi documentation <https://github.com/ecmwf/anemoi-docs>`_,
    package-level documentation), and the relevant section (e.g., user
    guide, getting started, or developer guide).
 
--  Consider maintainability — are interfaces still evolving?
-
--  Write clear and concise documentation using simple, direct sentences.
+-  You find the documentation in the `docs` directory of the Anemoi
+   repository. The documentation is written in reStructuredText (reST)
+   format. Add your changes to the appropriate file in the `docs`
+   directory and pay attention to the following:
 
--  Add references to related sections or external documentation where
-   applicable.
+   -  Consider maintainability — are interfaces still evolving?
+   -  Write clear and concise documentation using simple, direct
+      sentences.
+   -  Add references to related sections or external documentation where
+      applicable.
```

Comment:
Could we worth also mentioning that any plots or diagrams should go under the _`static` folder